dopamine/discrete_domains/atari_lib.py

```diff
@@ -48,7 +48,7 @@
 
 
 @gin.configurable
-def create_atari_environment(game_name=None, sticky_actions=True):
+def create_atari_environment(game_name=None, game_mode=0, game_difficulty=0, sticky_actions=True):
```

Comment:
Please keep default parameters in the same order to maintain backwards compatibility.